plugins/inputs/procstat/spec_processor.go

```diff
@@ -102,7 +102,7 @@ func (p *SpecProcessor) pushMetrics() error {
 		fields[prefix+"cpu_time_guest_nice"] = cpu_time.GuestNice
 	}
 
-	cpu_perc, err := p.proc.Percent(time.Duration(0))
+	cpu_perc, err := p.proc.Percent(time.Duration(3000 * time.Millisecond))
```

Comment:
What's the reason for this change? And why 3s, vs some other value, such as 1s, or 1ms?
This change will have the side effect that all process data gathered after it will be taken 3 seconds after all data before it. This can lead to bad/confusing data as 3 seconds is a long time.
This will also cause problems if someone has a gather interval shorter than 3 seconds.